templates/base.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE "html">
+<html lang="en">
+<head>
+  {% block title %}
+    <title>Taxi Service</title>
+  {% endblock %}
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta charset="usf-8">

```

Comment:
There is a typo in the charset meta tag. It should be `utf-8` instead of `usf-8`. This could cause issues with character encoding on your website.